server/src/main/java/org/apache/uniffle/server/ShuffleFlushManager.java

```diff
@@ -81,40 +82,10 @@ public ShuffleFlushManager(ShuffleServerConf shuffleServerConf, String shuffleSe
     this.maxConcurrencyOfSingleOnePartition =
         shuffleServerConf.get(ShuffleServerConf.SERVER_MAX_CONCURRENCY_OF_ONE_PARTITION);
 
-    int waitQueueSize = shuffleServerConf.getInteger(
-        ShuffleServerConf.SERVER_FLUSH_THREAD_POOL_QUEUE_SIZE);
-    BlockingQueue<Runnable> waitQueue = Queues.newLinkedBlockingQueue(waitQueueSize);
-    int poolSize = shuffleServerConf.getInteger(ShuffleServerConf.SERVER_FLUSH_THREAD_POOL_SIZE);
-    long keepAliveTime = shuffleServerConf.getLong(ShuffleServerConf.SERVER_FLUSH_THREAD_ALIVE);
-    threadPoolExecutor = new ThreadPoolExecutor(poolSize, poolSize, keepAliveTime, TimeUnit.SECONDS, waitQueue,
-        ThreadUtils.getThreadFactory("FlushEventThreadPool"));
+    threadPoolExecutor = createFlushEventExecutor();
+    startEventProcessor();
```

Comment:
I think line L85-86 should be placed after line L87-88.
Because `storageBasePaths` is accessed in `processEvent() -> flushToFile()`

Reply:
Done